asciidoc/volume2/discovery-scopes/tf2-ch-a-mdpws-discovery-scopes.adoc

```diff
@@ -30,7 +30,7 @@ URIs of a <<term_manufacturer>> name and serial number:
 ====
 ****
 
-.<<ref_iso_iec_14977_1996, Extended Backus-Naur Form>> for the encoding of production specifications

```

Comment:
Don't be confused. I had to remove the links here as otherwise weird escaping ensues. This happens because we had to disable anchors in captions (i.e. remove HTML tags). This seems to be somehow broken in Asciidoc or at least in the plugin system of Asciidoc.